fab/fabfile.py

```diff
@@ -692,6 +694,14 @@ def copy_tf_localsettings():
         ))
 
 
+@task
+@roles(ROLES_TOUCHFORMS)
+def build_formsplayer():
+    spring_dir = '{}/{}'.format(env.code_root, 'submodules/formsplayer')
+    with cd(spring_dir):
+        sudo('./gradlew build')
```

Comment:
How much time will this add to the build? Can we not setup a build on jenkins and then pull the latest? One other thought... is it necessary to have this coupled with the HQ deploy? I suppose decoupling it makes keeping track of things a bit more difficult.